base/applications/cmdutils/label/lang/pl-PL.rc

```diff
@@ -5,23 +5,24 @@
  *    http://www.reactos.org
  *    IRC: irc.freenode.net #reactos-pl;
  *    UTF-8 conversion by Caemyr (May, 2011)
+ *    Updated by pithwz - Piotr Hetnarowicz (piotrhwz@gmail.com) (April, 2020)
  */
 
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
 STRINGTABLE
 BEGIN
-//    STRING_LABEL_HELP "Wyświetla lub zmienia etykietę napędu.\n\nLABEL [napęd:][etykieta]\n"
-    STRING_LABEL_HELP "Creates, changes or deletes the volume label of a disk.\n\n\
-LABEL [drive:][label]\n\n\
-  drive:          Specifies the drive letter of a drive.\n\
-  label           Specifies the label of the drive."
+//    STRING_LABEL_HELP "Wyświetla lub zmienia etykietę woluminu dysku.\n\nLABEL [dysk:][etykieta]\n"
```

Comment:
What this comment was left for?

Reply:
this commment was in original file & I dont deleted him